src/main/kotlin/lotto/controller/LottoController.kt

```diff
@@ -37,6 +39,12 @@ class LottoController(private val lottoService: LottoService) {
         return Lotto(InputParser.parseWinningNumbers(readLine()!!))
     }
 
+    private fun getBonusNumber(): LottoNumber {
+        LottoGameView.printBonusNumber()
+
+        return InputParser.parseBonusNumber(readLine()!!)
```

Comment:
`requireNotNull`를 활용해보시는 것도 좋을것 같아요~
```suggestion
        return Lotto(InputParser.parseWinningNumbers(requireNotNull(readLine())))
```